src/main/java/pl/poznan/put/jsontools/logic/JsonFinalTransform.java

```diff
@@ -0,0 +1,7 @@
+package pl.poznan.put.jsontools.logic;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface JsonFinalTransform {
+    String execute(JsonNode json);
```

Comment:
There should be one interface in the Decorator design pattern and it would be convenient if it was this one.
https://refactoring.guru/design-patterns/decorator